packages/chart/src/CdcChart.tsx

```diff
@@ -1139,9 +1144,17 @@ export default function CdcChart({ configUrl, config: configObj, isEditor = fals
               <SkipTo skipId={handleChartTabbing} skipMessage='Skip Over Chart Container' />
               {/* Visualization */}
               {config?.introText && config.visualizationType !== 'Spark Line' && <section className='introText'>{parse(config.introText)}</section>}
-              <div className={getChartWrapperClasses().join(' ')}>
-                {/* All charts except sparkline */}
-                {config.visualizationType !== 'Spark Line' && chartComponents[config.visualizationType]}
+
+              <div style={{ marginBottom: computeMarginBottom(config, legend, currentViewport) }} className={getChartWrapperClasses().join(' ')}>
+                {/* All charts except line and sparkline */}
+                {config.visualizationType !== 'Spark Line' && config.visualizationType !== 'Line' && chartComponents[config.visualizationType]}
+
+                {/* Line Chart */}
+                {config.displayLineGraphAs === 'Line' && config.visualizationType === 'Line' && chartComponents['Line']}
+
+                {/* Line Chart to Bar Chart */}
+                {config.displayLineGraphAs === 'Bar' && config.visualizationType === 'Line' && ((config.visualizationSubType = 'Regular'), (config.barStyle = 'flat'), (config.isLollipopChart = false), chartComponents['Bar'])}
```

Comment:
Do not set config directly

Reply:
I've tried to update the config in  updateConfig() and loadConfig(). But when I remove these direct config settings, it causes the chart to not switch from line to bar or bar to line.